src/server/frontend_wayland/wl_seat.cpp

```diff
@@ -211,6 +211,8 @@ void mf::WlSeat::notify_focus(wl_client *focus)
         focused_client = focus;
         for (auto const listener : focus_listeners)
             listener->focus_on(focus);
+
+        executor->spawn([this] { resync_keyboard_state(); });
```

Comment:
It looks to me like this function is already on the Wayland thread. Why is the executor used? Is there a better way?